tests/test_aktualizr_kill.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+import logging
+import argparse
+
+from os import getcwd, chdir
+
+from test_fixtures import with_aktualizr, with_uptane_backend, KeyStore, with_secondary, with_path,\
+    DownloadInterruptionHandler, MalformedJsonHandler, with_director, with_imagerepo, InstallManager,\
```

Comment:
*Handler and with_treehub and InstallManager and with_imagerepo can be removed

Reply:
thanks!